src/playlist-loader.js

```diff
@@ -10,6 +10,9 @@ import videojs from 'video.js';
 import { Parser as M3u8Parser } from 'm3u8-parser';
 import window from 'global/window';
 
+// TODO: REMOVE
```

Comment:
Might be good to add the condition for removal (for future explorers).

Reply:
This line is going to be removed in this PR, I just have the stub here so the tests execute as much as possible